### DISCUSSION TITLE
What different with VmmemWSA & Windows Subsystem for Android

### DISCUSSION THREAD

**Discussion by Mhdfirmus** [microsoft/WSA#131] [General]
What different with VmmemWSA & Windows Subsystem for Android? both of that using high CPU usage, is that normal?
![image](https://user-images.githubusercontent.com/78803559/201904143-9e9e98a7-d4bd-4060-9be8-1b83af2b60d3.png)

**Comment by ghost** [comment]
You have to fully expand the Name column to see its complete name. if it says settings at the end then you probably opened WSA Settings. I see similar usage on my PC too but it all depends on the apps you install in Android and whether they are running or not, or if they running in the background like mobile apps usually do.

also you haven't posted your PC specs and exactly what apps you've installed. so can't say any more than that.

if CPU is weak or old then higher CPU usage is very normal.
